frontend/public/components/modals/disable-application-modal.tsx

```diff
@@ -14,39 +14,41 @@ export class DisableApplicationModal extends PromiseComponent {
 
   constructor(public props: DisableApplicationModalProps) {
     super(props);
-    this.state.cascadeDelete = true;
+    this.state.deleteCSV = true;
   }
 
   private submit(event): void {
     event.preventDefault();
 
     const {subscription, k8sKill} = this.props;
-    const deleteOptions = this.state.cascadeDelete ? {kind: 'DeleteOptions', apiVersion: 'v1', propagationPolicy: 'Foreground'} : null;
+    const deleteOptions = {kind: 'DeleteOptions', apiVersion: 'v1', propagationPolicy: 'Foreground'};
     const killPromises = [k8sKill(SubscriptionModel, subscription, {}, deleteOptions)]
-      .concat(_.get(this.props.subscription, 'status.installedCSV')
```

Comment:
I'm a little confused by the existing logic here. Won't concat always return a truthy value? I don't think the false condition of the ternary would have ever been reached?

Reply:
This is the common pattern I use to conditionally add an element to an array without a `let` expression. [This unit test](https://github.com/openshift/console/pull/489/files#diff-5e050b85e7ce6e4e74d751603be004f8R70) confirms that it works.

Reply:
Sorry I missed that the ternary was on the return from _.get not _concat